setup.cfg

```diff
@@ -20,6 +20,7 @@ install_requires =
     flask-session==0.8.0
     flask==3.1.2
     gunicorn==20.1.0
+    msgpack==1.1.2
```

Comment:
this section should really only be top-level dependencies, otherwise

from the flask-session docs, msgpack will become a transient dependency of flask-session in v1.0.0

it would probably be better to upgrade flask-session to v1.0+, then rebuild requirements.*txt with pip-compile.